atom/common/mac/main_application_bundle.mm

```diff
@@ -3,15 +3,15 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE-CHROMIUM file.
 
-#import "brightray/common/mac/main_application_bundle.h"
+#import "atom/common/mac/main_application_bundle.h"
 
 #include "base/files/file_path.h"
 #include "base/mac/bundle_locations.h"
 #include "base/mac/foundation_util.h"
 #include "base/path_service.h"
 #include "base/strings/string_util.h"
 
-namespace brightray {
+namespace atom {
 
 namespace {
 
```

Comment:
A possible TODO for future, could rewrite the check for Helper.app path with `base::mac::IsBackgroundOnlyProcess`